tools/benchmark/src/main/java/com/datadog/benchmark/internal/BenchmarkSpanToSpanEventMapper.kt

```diff
@@ -0,0 +1,60 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.benchmark.internal
+
+import com.datadog.benchmark.internal.model.BenchmarkContext
+import com.datadog.benchmark.internal.model.SpanEvent
+import io.opentelemetry.api.common.AttributeKey
+import io.opentelemetry.sdk.trace.data.SpanData
+
+internal class BenchmarkSpanToSpanEventMapper {
+
+    fun map(
+        context: BenchmarkContext,
+        spanData: SpanData
+    ): SpanEvent {
+        val durationNano = spanData.endEpochNanos - spanData.startEpochNanos
```

Comment:
```suggestion
        val durationNanos = spanData.endEpochNanos - spanData.startEpochNanos
```

to be consistent with naming